__tests__/app.test.js

```diff
@@ -94,27 +93,71 @@ describe("/api/articles/:article_id", () => {
-            expect(response.body.articles.length).toBe(13)
-            response.body.articles.forEach((article) => {
-            expect(article).toMatchObject({
-                author: expect.any(String),
-                title: expect.any(String),
-                article_id: expect.any(Number),
-                topic: expect.any(String),
-                created_at: expect.any(String),
-                votes: expect.any(Number),
-                article_img_url: expect.any(String),
-                comment_count: expect.any(Number)
-              });
+describe("/api/articles", () => {
+  it("GET responds with status code 200", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  it("GET responds with slug and description fields", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        expect(response.body.articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        expect(response.body.articles.length).toBe(13);
+        response.body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+      });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  it("GET responds with status code 200", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        console.log(response.body.comments)
+        expect(response.body.comments).toBeSortedBy("created_at", {
+          descending: true,
+        })
+        expect(response.body.comments.length).toBe(11);
+        expect(response.body.comments[0].article_id).toBe(1);
+        response.body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
         });
-    });
-  })});
\ No newline at end of file
+      });
+  });
+  it("Reponds 400 if invalid article id is provided", () => {
+    return request(app)
+      .get("/api/articles/cat/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toEqual("bad request");
+      });
+  });
+  it("Responds with 'does not exist' if valid article ID provided that does not exist", () => {
+    return request(app)
+      .get("/api/articles/345/comments")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.err).toBe("Article does not exist");
+      });
+  });
+});
```

Comment:
What does your endpoint return when an article that has no comments is requested?